### PR TITLE
Add Sitemap Options

### PR DESCRIPTION
**Pull request by codedre**
Add the 3 optional tags base on [sitemap.org](http://www.sitemaps.org/protocol.html) schema

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,20 @@ Add to your webpack config -- see below for an example. The plugin signature is:
 * `paths` is the array of locations on your site -- this can be the same one you pass to `static-site-generator-webpack-plugin`
 * `filename` is the name of the output file -- the default is `sitemap.xml`
 
+### Options
+
+* `lastMod` [boolean] default `false`
+* `priority` [number] defualt `null`
```
Comment (schneidmaster) [inline README.md]: Typo here -- "defualt" -> "default"

File: README.md [hunk 2]
```diff
@@ -16,6 +16,20 @@ Add to your webpack config -- see below for an example. The plugin signature is:
...
+* `changeFreq` [string] default `null`, list of applicable values based on [sitemaps.org]('http://www.sitemaps.org/protocol.html')
```
Comment (schneidmaster) [inline README.md]: Need to remove the single quotes around the sitemaps.org URL for it to be properly linked in Markdown

**Comment by codedre** [comment]
Congratulations on getting married!!!!

**Comment by schneidmaster** [comment]
Thanks! Changes look good, cheers!

**Pull request merged**